code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -212,6 +214,8 @@
 	icon_living = "snowlegion_head"
 	icon_aggro = "snowlegion_head"
 	icon_dead = "snowlegion_head"
+	snow_legion = TRUE
+	can_infest_dead = TRUE
```

Comment:
pretty sure this is already set on normal legion

Reply:
it isn't for some reason